### PR TITLE
Add missing zh-rTW translation and improve previous ones

### PR DESCRIPTION
**Pull request by brchiu**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [*] Run the unit tests with `gradlew app:connectedGmsDebugAndroidTest` to make sure you didn't break anything (with an emulator running or phone connected)

- [*] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -36,10 +38,12 @@
     <string name="show_grid_pref">赤经赤纬网格</string>
     <string name="show_horizon_pref">水平线</string>
     <string name="show_sky_gradient">天空梯度</string>
+    <string name="show_meteors_pref">Meteor Showers</string>
```
Comment (jaydeetay) [inline app/src/main/res/values-zh-rCN/strings.xml]: This shouldn't be needed - in the absence of a translation the app should already fall back to the English string.

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 2]
```diff
@@ -134,11 +146,13 @@
     <string name="hdf" translation_description="hubble object">哈伯深場影像</string>
     <string name="rho_oph" translation_description="hubble object">心宿增四星</string>
     <string name="menu_tos">服務條款</string>
+    <string name="menu_diagnostics">Diagnostics</string>
```
Comment (jaydeetay) [inline app/src/main/res/values-zh-rTW/strings.xml]: This shouldn't be needed - if we don't have a Chinese translation for this text it should already fall back to English.  Did you mean to provide a translation here?

Comment (brchiu) [reply app/src/main/res/values-zh-rTW/strings.xml]: @jaydeetay, I will correct it and add new commit to this pull request. thank you.

**Comment by brchiu** [comment]
@jaydeetay , I mainly focus on translation of Traditional Chinese which is used in our place, as for Simplified Chinese translation, I would prefer let people who are more acquainted with it to fix due to they might have their own wording.

I copied strings from English `string.xml` to Traditional Chinese directory is to remind me or any other volunteer that there is remaining strings to be translated. I know they will fall back to English automatically even we don't copy them. Thanks for your kindly explanation.

**Comment by brchiu** [comment]
hi @jaydeetay , I notice from github mail list that some recent pull requests seems not use string id for text but use hard coded text, it might make us difficult to do translation for them in the future. If possible, could you also remind those PR authors to use string id instead ? thank you very much.

**Comment by jaydeetay** [comment]
Thanks again for this! Yes, I will do my best to spot PRs that don't account for localization.

**Pull request merged**